core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,10 +39,35 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
```

Comment:
I think we're making two unsafe assumptions here:
- The user understands where we draw the distinction between Alluxio and Non-Alluxio errors.
- `IOException` always means Non-Alluxio error and non-`IOException` always means Alluxio error.

I like Jiri's proposal of using
```java
("Exit (Error): {}", callable, e)
```
to indicate that an error has occurred.